my-app/src/app/(dashboard)/agents/[agentId]/error.tsx

```
@@ -0,0 +1,11 @@
+"use client"
+
+import { Error } from "@/components/error"
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)